client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import { Box, Grid, Typography, Paper, CircularProgress } from '@mui/material';
+import useStyles from './useStyles';
+import RequestInfo from '../../components/RequestInfo/RequestInfo';
+import MyCalendar from '../../components/MyCalendar/MyCalendar';
+import { useAuth } from '../../context/useAuthContext';
+import getSitterRequests from '../../helpers/APICalls/sitterRequests';
+import { RequestApiData } from '../../interface/RequestApiData';
+import { format } from 'date-fns';
+import changeStatus from '../../helpers/APICalls/changeStatus';
+
+export default function ManageBookings(): JSX.Element {
+  const classes = useStyles();
+  const history = useHistory();
+  const { loggedInUser } = useAuth();
+  const { sitterRequests } = useAuth();
```

Comment:
Good thinking but we didn't need to store the bookings in the Auth context as they could've been stored in a local state within this component :)